bubuku/features/restart_if_dead.py

```diff
@@ -45,12 +50,19 @@ def __init__(self, broker: BrokerManager, zk: BukuExhibitor):
         super().__init__()
         self.broker = broker
         self.zk = zk
+        self.need_check = True
 
     def check(self) -> Change:
+        if not self.need_check:
+            return None
         if self.broker.is_running_and_registered():
             return None
         _LOG.info('Oops! Broker is dead, triggering restart')
-        return StartBrokerChange(self.broker, self.zk)
+        self.need_check = False
+        return StartBrokerChange(self.broker, self.zk, self.on_check_removed)
+
+    def on_check_removed(self):
+        self.need_check = True
```

Comment:
Let's suppose, that we are starting bubuku on several nodes (10.0.0.1, 10.0.0.2) at the same time.
After frist iteration there will be to changes in queue:
[{'start':'10.0.0.1'}, {'start':'10.0.0.2'}].
But only one of them will be applied (let's suppose that it's on 10.0.0.1). 
That means that during 'start' on 10.0.0.1 check on 10.0.0.2 will be executed several times, and each time it will say: 'broker is not started, add new change'. 
I don't want this kind of situation, so I allow only one StartChange for each instance in change queue.